### DISCUSSION TITLE
Meeting: Celo Wallet and (d)App Council #8 [March] 💬

### DISCUSSION THREAD

**Discussion by arthurgousset** [celo-org/celo-monorepo#9304] [Wallet and (d)App Council]
### Join
Google Meet: [meet.google.com/uwj-wfse-uzm](https://meet.google.com/uwj-wfse-uzm)
Recording: See in comments👇

### Useful links

- Notion page with meeting notes: [Celo Wallet Council Meeting Notes](https://www.notion.so/celoplatform/Celo-Wallet-Council-61dfe28e98a543409cf6c181b132c4de)
- Last meeting: [Celo Wallet and (d)App Council #7 [February]](https://github.com/celo-org/celo-proposals/issues/308)
- Forum post: [Celo Wallet and (d)App Council #8 [March]](https://forum.celo.org/t/celo-wallet-and-d-app-council-8-march/2980) 

--------------------------------------------

Hey all :wave:

The monthly Celo Wallet and (d)App Council meeting is taking place on **Wednesday, March 2nd (9am PDT/6pm CEST)**.

### :calendar: What is this?

- A *monthly* meeting for Celo **developers, founders,** **product** people (anyone really!) to share
  - wishes,
  - pain points, and
  - ideas

### 🗒️  Agenda

- Check out, vote, and add to the agenda below
- 🗳️ You can **upvote/downvote** agenda items below.
- ✍️ Please feel free to add agenda items (however small or big!) on Github. For example:

  - broad discussion topics to **amplify** your ideas/features/frictions,
  - requests for **feedback** (from the community),
  - roadmap **updates** ,
  - etc (anything that would be useful for you really!)

### :star_struck: How do I participate?

- Join using Google Meet: meet.google.com/uwj-wfse-uzm
- Take a peek at the [previous meeting notes](https://www.notion.so/celoplatform/Celo-Wallet-and-d-App-Council-monthly-61dfe28e98a543409cf6c181b132c4de)

See you there! :rocket:

--------------------------------------------

### Housekeeping:

- [x] Send update calendar invite with Github link
- [x] Add agenda items to vote on
- [x] Publicise on forum and Discord

--------------------------------------------

### Proposed agenda items 👇  

Please feel free to add agenda items (however small or big!) below.
 
For example:
- broad discussion topics to **amplify** your ideas/features/frictions, 
- requests for **feedback** (from the community), 
- roadmap **updates**,
- etc (anything that would be useful for you really!) 

🗳️ You can upvote/downvote agenda items below.

**Comment by vibern0** [comment]
I'd like to know what are the plans with https://github.com/celo-org/use-contractkit

**Reply by arthurgousset** [reply]
cc @dckesler (I suspect you might have useful context here)

**Comment by cajubelt** [comment]
We (Valora) would like to give a quick update on FiatConnect, a universal API specification for cash-in/cash-out providers to integrate with wallets

**Comment by aaronmgdr** [comment]
One thing ive been thinking about is a way for determining if a wallet supports setting feeCurrency. and if it does having a function from contractKit to choose the best feeCurrency for an account.

**Reply by arthurgousset** [reply]
This is a great one! Perhaps even from a UI perspective in an app, something like

> _This app support Celo native fee currencies_ ✅

This might reduce users' confusion when using MetaMask (for example) and fees are paid in Celo.

**Comment by isabellewei** [comment]
The cLabs identity team is working on a version 2 of Attestation Service that uses a model of centralized issuers and extends attestations to other offchain identifiers (ex. twitter handles or emails). The work is still in the design phase, but we would love to get a gauge of interest to loop into discussions

**Comment by aaronmgdr** [comment]
Discussion for use-ck fork reunifications here https://github.com/celo-org/use-contractkit/discussions/125

**Comment by arthurgousset** [comment]
Discussion 
- https://github.com/celo-org/celo-monorepo/discussions/9364

> in response to talks of contractkit's notorious bundle size

**Comment by arthurgousset** [comment]
Thanks to all who attended the recent Wallet and (d)App Council meeting. Here is the

- Meeting recording 🎥: https://drive.google.com/file/d/1cVbVd4cTilr6j7UM3VYiVfIg2e3TY-H8/view?usp=sharing
- Meeting notes 📝: https://www.notion.so/celoplatform/Celo-Wallet-and-d-App-Council-monthly-61dfe28e98a543409cf6c181b132c4de#ac7b0a87fc4544e8882ceb649aee5448

Link to next meeting: 
- https://github.com/celo-org/celo-monorepo/discussions/9365

**Comment by arthurgousset** [comment]
Reflecting Eric's request for better docs to integrate and using the Attestation Service: 
- https://github.com/celo-org/celo-monorepo/discussions/9366
